frontend/src/lib/constants.tsx

```diff
@@ -332,6 +332,10 @@ export const SESSION_REPLAY_MINIMUM_DURATION_OPTIONS: LemonSelectOptions<number
         label: '15',
         value: 15000,
     },
+    {
+        label: '30 (not recommended)',
```

Comment:
```suggestion
        label: '30',
```

i think this prompts the "why not?" question
when really we should move to a persistent queue and the confusion here goes away 🙈

Reply:
silly question... does the SDK validate this max at all 🙈